packages/replayio/src/utils/recordings/removeFromDisk.ts

```diff
@@ -15,10 +15,14 @@ export function removeFromDisk(id?: string) {
     if (recording) {
       const { metadata, path } = recording;
 
-      metadata.sourcemaps?.forEach(path => {
-        debug("Removing recording source-map file %s", path);
+      metadata.sourcemaps.forEach(sourceMap => {
+        debug("Removing recording source-map file %s", sourceMap.path);
+        removeSync(sourceMap.path);
```

Comment:
This will remove source maps and original sources even if their uploads failed. IIRC re-uploading the same recording isn’t a thing (please correct me if i’m wrong). So would there be any value in keeping them around in such a case?